### DISCUSSION TITLE
My Instagram account has been hacked

### DISCUSSION THREAD

**Discussion by Vaibhaarti** [Authenticator-Extension/Authenticator#1279] [General]
hi team,

my instagram account has been hacked. the instagram team helped me to reset the password which also been done but unable to login as authentication code is not working. i have tried my backup code too but that too also not working.

it seems like my athenticator also been hached 
.
pleaes help me on urgent basis

**Comment by khanhina143** [comment]
Instagram hack

**Reply by brarj9421-create** [reply]
Jobanbrar84 account hack

**Reply by lucasgrinvall-wq** [reply]
For account recovery process like Gmail, Facebook, instagram and other social media recovery account, contact : ( Hackertechs001@gmail.com ) . He will help you. Ensure you’ve access to the number and recovery email linked to the account. 
https://t.me/hackertechs001
They offer proof on whatsapp hacking, reach out to them on my through mail or telegram.
Facebook password stealing, bitcoin recovery ,lost crypto recovery ,
They can monitor any iphone and android , 
i know alot of scammers are out there ,they  offer legit services with money back gurrantee if things dont work out.

**Comment by himanshuatal706-netizen** [comment]
Heyy GitHub pls my instagram account recovery my instagram account disabled without reason please check my account 

E-mail : himanshuatal706@gmail.com 
Instagram account: himanshuu__.00__________.5
Mobile 9772322565

**Reply by bendokhaneakram09-stack** [reply]
himanshuu__.00__________.5

**Reply by bendokhaneakram09-stack** [reply]
hacker.tunisia_09

**Reply by lucasgrinvall-wq** [reply]
For account recovery process like Gmail, Facebook, instagram and other social media recovery account, contact : ( Hackertechs001@gmail.com ) . He will help you. Ensure you’ve access to the number and recovery email linked to the account. 
https://t.me/hackertechs001
They offer proof on whatsapp hacking, reach out to them on my through mail or telegram.
Facebook password stealing, bitcoin recovery ,lost crypto recovery ,
They can monitor any iphone and android , 
i know alot of scammers are out there ,they  offer legit services with money back gurrantee if things dont work out.

**Comment by himanshuatal706-netizen** [comment]
Hello Team Support,

My Instagram account was disabled without any clear reason. I believe this may have been a mistake. I respectfully request you to review my account again.

Account details:
Email: [himanshuatal706@gmail.com]

Instagram Username: himanshuu__.00__________.5
Mobile Number: 9587891343

I have always tried to follow Instagram’s Community Guidelines and Terms of Use. If there was any unintentional violation, I apologize and request another chance.

Please review and restore my account as soon as possible.

Thank you,
Himanshu
<img width="720" height="1600" alt="Screenshot_20260521-220803" src="https://github.com/user-attachments/assets/9ff46c2a-3215-4253-802d-8acce2702dd7" />

**Reply by bendokhaneakram09-stack** [reply]
on Aug 26, 2024 · 3 comments · 3 replies

Vaibhaarti
on Aug 26, 2024
hi team,

my instagram account has been hacked. the instagram team helped me to reset the password which also been done but unable to login as authentication code is not working. i have tried my backup code too but that too also not working.

it seems like my athenticator also been hached
.
pleaes help me on urgent basis

Replies:3 comments · 3 replies

khanhina143
on May 12
Instagram hack

1 reply
@brarj9421-create
brarj9421-create
3 weeks ago
Jobanbrar84 account hack


himanshuatal706-netizen
on May 21
Heyy GitHub pls my instagram account recovery my instagram account disabled without reason please check my account

E-mail : himanshuatal706@gmail.com
Instagram account: himanshuu__.00__________.5
Mobile 9772322565

2 replies 1 new
@bendokhaneakram09-stack
bendokhaneakram09-stack
1 minute ago
himanshuu__.00__________.5

@bendokhaneakram09-stack
bendokhaneakram09-stack
now
hacker.tunisia_09


himanshuatal706-netizen
on May 21
Hello Team Support,

My Instagram account was disabled without any clear reason. I believe this may have been a mistake. I respectfully request you to review my account again.

Account details:
Email: [akrambendokhane16@gmail.com]

Instagram Username: hacker.tunisia_09
Mobile Number: 22590439

I have always tried to follow Instagram’s Community Guidelines and Terms of Use. If there was any unintentional violation, I apologize and request another chance.

Please review and restore my account as soon as possible.

Thank you,

**Reply by lucasgrinvall-wq** [reply]
> Hello Team Support,
> 
> My Instagram account was disabled without any clear reason. I believe this may have been a mistake. I respectfully request you to review my account again.
> 
> Account details: Email: [[himanshuatal706@gmail.com](mailto:himanshuatal706@gmail.com)]
> 
> Instagram Username: himanshuu__.00__________.5 Mobile Number: [9587891343](tel:9587891343)
> 
> I have always tried to follow Instagram’s Community Guidelines and Terms of Use. If there was any unintentional violation, I apologize and request another chance.
> 
> Please review and restore my account as soon as possible.
> 
> Thank you, Himanshu <img alt="Screenshot_20260521-220803" width="720" height="1600" src="https://private-user-images.githubusercontent.com/286731073/596238216-9ff46c2a-3215-4253-802d-8acce2702dd7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODk3NTc4MzYsIm5iZiI6MTc4OTc1NzUzNiwicGF0aCI6Ii8yODY3MzEwNzMvNTk2MjM4MjE2LTlmZjQ2YzJhLTMyMTUtNDI1My04MDJkLThhY2NlMjcwMmRkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDkxOFQxODUyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDBjY2Y3MmExZTdhNjFjMzUzNzk0ODk5NmMxYzM5MTEzYjdlMDMyNWI5MzNiM2ZiODIzNWYzYjQ4OThlYjQxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.M532YWN--HV6WIaPFjMu5TSGLmQ0r9lt-ru_VUfXaAM">

“For account recovery process like Gmail, Facebook, instagram and other social media recovery account, contact : ( Hackertechs001@gmail.com ) . He will help you. Ensure you’ve access to the number and recovery email linked to the account. 
https://t.me/hackertechs001
They offer proof on whatsapp hacking, reach out to them on my through mail or telegram.
Facebook password stealing, bitcoin recovery ,lost crypto recovery ,
They can monitor any iphone and android , 
i know alot of scammers are out there ,they  offer legit services with money back gurrantee if things dont work out”

**Comment by lucasgrinvall-wq** [comment]
For account recovery process like Gmail, Facebook, instagram and other social media recovery account, contact : ( Hackertechs001@gmail.com ) . He will help you. Ensure you’ve access to the number and recovery email linked to the account. 
https://t.me/hackertechs001
They offer proof on whatsapp hacking, reach out to them on my through mail or telegram.
Facebook password stealing, bitcoin recovery ,lost crypto recovery ,
They can monitor any iphone and android , 
i know alot of scammers are out there ,they  offer legit services with money back gurrantee if things dont work out.
